website_sale_one_step_checkout/controllers/main.py

```diff
@@ -0,0 +1,224 @@
+# -*- coding: utf-8 -*-
```

Comment:
In general check the flake8 issues.
- main.py:6:1: I100 Imports statements are in the wrong order. from odoo should be before from odoo.http
- main.py:6:1: I101 Imported names are in the wrong order. Should be _, http
- main.py:7:1: I201 Missing newline before sections or imports.
- main.py:60:80: E501 line too long (81 > 79 characters)
- main.py:74:80: E501 line too long (80 > 79 characters)
- main.py:143:80: E501 line too long (80 > 79 characters)